api/v1beta1/tobiko_types.go

```diff
@@ -28,6 +29,11 @@ import (
 type TobikoSpec struct {
 	CommonOptions `json:",inline"`
 
+	// The desired amount of resources that should be assigned to each test pod
+	// spawned using the Tobiko CR. https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements
+	// +kubebuilder:default:={limits: {cpu: "4000m", memory: "4Gi"}, requests: {cpu: "2000m", memory: "2Gi"}}
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
@eduolivares , can you please 🙏 suggest the correct Resource Limits for the tobiko test pods? It will be possible to override the default value in the Tobiko CR. For example, in Tempest the amount of resources which is needed is dependent on the number of workers that are used. Maybe it is the same with Tobiko as well?

Reply:
Discussed with Eduardo in PM and we agreed on:
1. Setting the default limits to:
```
kubebuilder:default:={limits: {cpu: "8000m", memory: "8Gi"}, requests: {cpu: "4000m", memory: "4Gi"}}
```

2. And setting of the default value of `NumProcesses` to `4`.